src/NServiceBus.Core/Routing/RoutingFeature.cs

```diff
@@ -125,6 +127,7 @@ static string LocalAddress(IBuilder builder)
             return builder.Build<ReadOnlySettings>().LocalAddress();
         }
 
+        /// Not sure about this one
```

Comment:
I think we should invert the dependencies and move this code to place where `ISubscriptionStorage` is registered.